cmd/hope/deploy.go

```diff
@@ -252,6 +253,62 @@ var deployCmd = &cobra.Command{
 				if err := kubeutil.ExecKubectl(kubectl, allArgs...); err != nil {
 					return err
 				}
+			case hope.ResourceTypeHelm:
+				haveRepo := false
+				currentRepos, err := helm.GetHelm("repo", "list")
+				if err != nil {
+					return err
+				}
+
+				for _, repoLine := range strings.Split(currentRepos, "\n") {
```

Comment:
This should probably be dumped into the `helm` pkg, or at the very least, some of what this is doing.